horizon/Distribution.py

```diff
@@ -45,6 +45,10 @@ class Distribution(BaseModel):
         If IANA Media Type is available mediaType property should be used.
     mediaType: str
         Must be from https://www.iana.org/assignments/media-types/media-types.xhtml
+    conformsTo: HttpUrl
+        An established standard to which the distribution conforms. 
+        Used to indicate the model, schema, ontology, view or profile that this 
+        representation of a dataset conforms to.
```

Comment:
Let's add one specific example here. Something like:
"https://project-open-data.cio.gov/v1.1/schema"

Reply:
With an example this is good and ready to merge.

Reply:
Perfect!